docs/delivery_methods/ios.md

```diff
@@ -28,7 +28,7 @@ With custom configuration:
 
 ```ruby
 class CommentNotification
-  deliver_by :ios, format: :ios_format, cert_path: :ios_cert_path, key_id: :ios_key_id, team_id: :ios_team_id, pool_size: 5
+  deliver_by :ios, format: :ios_format, development: :development?, pool_size: 5
```

Comment:
I use this `development?` on almost every project I work on so I think it is worth calling out here in the docs.

The two customizations were removed because they don't provide more value, in my opinion, over `#ios_format` customized above.

Reply:
You can probably just do this inline since the Rails env will not change.

```suggestion
  deliver_by :ios, format: :ios_format, development: Rails.env.local?, pool_size: 5
```